tests/integration-service/namespace-backed-environments.go

```diff
@@ -59,6 +72,15 @@ var _ = framework.IntegrationServiceSuiteDescribe("Namespace-backed Environment
 
 			integrationTestScenario, err = f.AsKubeAdmin.IntegrationController.CreateIntegrationTestScenarioWithEnvironment(applicationName, testNamespace, gitURLForNBE, revisionForNBE, pathInRepoForNBE, userPickedEnvironment)
 			Expect(err).ShouldNot(HaveOccurred())
+			phaseDTC = "Bound"
+			phaseDT = "Bound"
+
+			consoleRoute, err := f.AsKubeAdmin.CommonController.GetOpenshiftRoute("console", "openshift-console")
+			Expect(err).ShouldNot(HaveOccurred())
+
+			if utils.IsPrivateHostname(consoleRoute.Spec.Host) {
+				Skip("Using private cluster (not reachable from Github), skipping...")
```

Comment:
Why are we skipping tests here?

Reply:
This test errors out in case of private cluster, hence the skipping.